lib/redis.js

```diff
@@ -91,8 +91,9 @@ module.exports = CoreObject.extend({
     });
   },
 
-  activeRevision: function(keyPrefix) {
-    var currentKey = keyPrefix + ':current';
+  activeRevision: function(keyPrefix, activationSuffix) {
+    activationSuffix = activationSuffix || 'current';
```

Comment:
given that it already has a default in `index.js` do we need to duplicate it here?